ui/src/pages/namespace/Explorer/Workflow/Overview/Content.tsx

```diff
@@ -0,0 +1,35 @@
+import Donut from "./Donut";
+import { MetricsResponseSchemaType } from "~/api/metrics/schema";
+import { NoResult } from "~/design/Table";
+import { PieChart } from "lucide-react";
+import { getDonutConfig } from "./utils";
+import { useTranslation } from "react-i18next";
+
+const Content = ({
+  isFetched,
+  data,
+}: {
+  isFetched: boolean;
+  data: MetricsResponseSchemaType;
+}) => {
+  const { t } = useTranslation();
+  if (!isFetched || !data?.data)
```

Comment:
If `data.data` is undefined, this would technically not be a loading state, right? Because it would now display the loading text. 

The schema currently is 
```typescript
export const MetricsResponseSchema = z.object({
  data: MetricsObjectSchema.optional(),
});
``` 
and I wonder if we can update the schema and change `data.data` as non-optional and just remove the `|| !data?.data`? I tried this locally and could not produce a situation where `data.data` is `null`

Reply:
You're right, we can make it non-optional and remove that check :-)